src/hubmap_translation/addl_index_transformations/portal/translate.py

```diff
@@ -44,30 +43,6 @@ def _map(doc, key, map):
             _map(ancestor, key, map)
 
 
-def _add_metadata_metadata_placeholder(doc):
```

Comment:
I think some folks still use the dev-search feature. You'll have to make an accompanying change to handle https://github.com/hubmapconsortium/portal-ui/blob/0cb4529dcf06ccfb05317f7ed9891291be002a77/context/app/static/js/pages/search/DevSearch.jsx#L87.

Reply:
Got it!

Reply:
Good point @john-conroy.  Personally I use that `/dev-search` endpoint for reindex verification.